src/protocol/basics/share_known_value/mod.rs

```diff
@@ -0,0 +1,41 @@
+use crate::bits::Fp2Array;
```

Comment:
I know that we've done this before, but I don't like it when we make a module that is a single mod.rs in a directory.  It's harder to find.